packages/drivers/odsp-driver/src/createFile.ts

```diff
@@ -99,7 +99,8 @@ export async function createNewEmptyFluidFile(
     epochTracker: EpochTracker,
 ): Promise<string> {
     const filePath = newFileInfo.filePath ? encodeURIComponent(`/${newFileInfo.filePath}`) : "";
-    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}`);
+    // add .tmp extension to empty file (host is expected to rename)
+    const encodedFilename = encodeURIComponent(`${newFileInfo.filename}.tmp`);
```

Comment:
I wonder, could we fix the asymmetry/non-local reasoning by having two driver methods, like:

.beginCreateFluidFile() <- Creates file with '.tmp' extension
.endCreateFluidFile()    <- Renames file to remove .tmp

(Honest question, since I don't know a lot about the runtime/host/driver layering.)

Reply:
I don't think we have a way to rename files (maybe @vladsud can confirm), so we would need to talk to ODSP to do it this way.